packages/itwin/one-click-lca-widget/package.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "@itwin/one-click-lca-widget",
+  "version": "0.1.0",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the iTwin Carbon Calculation.",
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "Carbon Calculation"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/itwin/one-click-lca-widget"
```

Comment:
Can you update the link to `/imodeljs/viewer-components-react` -> `/itwin/viewer-components-react/`